.github/workflows/cron-checks.yml

```diff
@@ -19,13 +19,17 @@ jobs:
     name: Test E2E UI (Debug)
     strategy:
       matrix:
-        ios: [12.4, 13.7, 14.5, 15.4, latest]
-        device: ["iPhone 8"]
         include:
           - ios: latest
-            device: "iPhone SE (3rd generation)"
-          - ios: latest
-            device: "iPad Air (5th generation)"
+            device: "iPhone 14 Pro Max"
+          - ios: 15.4
+            device: "iPhone 8"
+          - ios: 14.5
+            device: "iPhone SE (2nd generation)"
+          - ios: 13.7
+            device: "iPad Air (3rd generation)"
+          - ios: 12.4
+            device: "iPhone X"
```

Comment:
Isn't this going to make the snapshot tests fail as they use different screen resolutions?

Reply:
This matrix is only for e2e tests 🙂

Reply:
🚀